README.md

```diff
@@ -163,9 +202,10 @@ rely on `gomplate` or `helm` to build chart packages.
 Here's a stub you can include into a `Makefile` to make it easier to install binary dependencies.
 
 ```
+THIS_PATH := $(abspath $(patsubst %/,%,$(dir $(abspath $(lastword $(MAKEFILE_LIST))))))
```

Comment:
Can you give some more context on why this is necessary? What problem did this solve for you?